src/raftstore/store/store.rs

```diff
@@ -2029,17 +2029,62 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         Ok(())
     }
 
+    fn check_msg_peer(&self, msg: &RaftCmdRequest) -> Result<&Peer> {
```

Comment:
Does this function need to be reused?

Reply:
It should be, but it can't, it relies on `Map<region_id, Peer>` which only `Store` has it.

Reply:
It returns a `Peer`, maybe `check_and_get_msg_peer`?

Reply:
maybe check_propose/request_peer? 

Msg is too common here.